xapian-applications/omega/handler_libebook.cc

```diff
@@ -0,0 +1,133 @@
+/** @file handler_libebook.cc
+ * @brief Extract text and metadata using libe-book.
+ */
+/* Copyright (C) 2019 Bruno Baruffaldi
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301
+ * USA
+ */
+#include <config.h>
+#include "handler.h"
+
+#include <xapian.h>
+#include <string.h>
+#include <unordered_map>
+#include <memory>
+#include <iostream>
+
+#include <librevenge-0.0/librevenge-generators/librevenge-generators.h>
+#include <librevenge-0.0/librevenge-stream/librevenge-stream.h>
```

Comment:
I don't think you should have `librevenge-0.0/` explicitly here.

```
$ pkg-config libe-book-0.1 --cflags
-I/usr/include/libe-book-0.1 -I/usr/include/libxml2 -I/usr/include/librevenge-0.0
```

So if you've hooked up those flags correctly in `Makefile.am` then the compiler should look in `/usr/include/librevenge-0.0`.  By hard coding this here, you'll stop things working if that changes to `librevenge-0.1` for example.

Reply:
Okay, i didn't notice that I was hard coding that. I will remove it.